e2e-tests/src/tests/bindings.test.ts

```diff
@@ -0,0 +1,638 @@
+/**
+ * Test the Action Bindings for Merlin and the Scheduler
+ *
+ * Health endpoints are already tested in health.test.ts
+ */
+import { expect, test } from '@playwright/test';
+import req, {awaitSimulation} from '../utilities/requests.js';
+import time from "../utilities/time.js";
+import * as urls from '../utilities/urls.js';
+
+test.describe('Merlin Bindings', () => {
+  let mission_model_id: number;
+  let plan_id: number;
+  let admin: User = {
+    username: "Admin_User",
+    default_role: "aerie_admin",
+    allowed_roles: ["aerie_admin"],
+    session: {'x-hasura-role': 'aerie_admin', 'x-hasura-user-id': 'Admin_User'}
+  };
+  let nonOwner: User = {
+    username: "not_owner",
+    default_role: "user",
+    allowed_roles: ["user", "viewer"],
+    session: {'x-hasura-role': 'user', 'x-hasura-user-id': 'not_owner'}
+  };
+
+  test.beforeAll(async ({ request }) => {
+    // Insert the users
+    await req.createUser(request, admin);
+    await req.createUser(request, nonOwner);
+
+    // Insert the Mission Model
+    let rd = Math.random()*100000;
+    let jar_id = await req.uploadJarFile(request);
+    const model: MissionModelInsertInput = {
+      jar_id,
+      mission: 'aerie_e2e_tests',
+      name: 'Banananation (e2e tests)',
+      version: rd + "",
+    };
+    mission_model_id = await req.createMissionModel(request, model);
+
+    const plan_start_timestamp = "2023-01-01T00:00:00+00:00";
+    const plan_end_timestamp = "2023-01-02T00:00:00+00:00";
+    // Insert the Plan
+    const plan_input : CreatePlanInput = {
+      model_id : mission_model_id,
+      name : 'test_plan_' + rd,
+      start_time : plan_start_timestamp,
+      duration : time.getIntervalFromDoyRange(plan_start_timestamp, plan_end_timestamp)
+    };
+    plan_id = await req.createPlan(request, plan_input, admin.session);
+  });
+  test.afterAll(async ({ request }) => {
+    // Remove Model and Plan
+    await req.deleteMissionModel(request, mission_model_id);
+    await req.deletePlan(request, plan_id);
+
+    // Remove Users
+    await req.deleteUser(request, admin.username);
+    await req.deleteUser(request, nonOwner.username);
+  });
+
+  // "resourceTypes" and "getActivityEffectiveArguments" are not tested, as they are deprecated
```

Comment:
Do we have a ticket to remove these deprecated endpoints?

Reply:
We don't

Reply:
Created one here: https://github.com/NASA-AMMOS/aerie/issues/1079